sqllogictest-bin/src/engines/postgres.rs

```diff
@@ -48,10 +49,18 @@ impl Drop for Postgres {
 impl sqllogictest::AsyncDB for Postgres {
     type Error = tokio_postgres::error::Error;
 
-    async fn run(&mut self, sql: &str) -> Result<String, Self::Error> {
-        use std::fmt::Write;
+    async fn run(&mut self, sql: &str) -> Result<DBOutput, Self::Error> {
+        let mut output = vec![];
+
+        let is_query_sql = {
+            let lower_sql = sql.to_ascii_lowercase();
```

Comment:
The input SQL may have leading whitespaces.
```suggestion
            let lower_sql = sql.trim_start().to_ascii_lowercase();
```